app/src/test/java/com/example/musicapp/InstrumentDataIntegrationTest.kt

```diff
@@ -13,46 +13,32 @@ import org.junit.Assert.*
 import org.junit.Rule
 import org.junit.rules.TestRule
 
-
-
 class InstrumentDataIntegrationTest {
 
     @get:Rule
-    var rule: TestRule =  InstantTaskExecutorRule()
-    lateinit var mvm:MainViewModel
-
+    var rule: TestRule = InstantTaskExecutorRule()
+    lateinit var mvm: MainViewModel
```

Comment:
Although it is optional to add space between properties, I took this space out to keep it consistent with other classes throughout your project (Not one of my code review suggestions).